tests/unit/algos/set/ArrayUtils.js

```diff
@@ -0,0 +1,12 @@
+const assert = require('assert');
+const symDiff = require('../../../../lib/algos/set/ArrayUtils').symDiff;
+
+describe('ArrayUtils', () => {
```

Comment:
```suggestion
describe('ArrayUtils.symDiff', () => {
```

Reply:
please ignore this if you're going to add more tests for the other helpers (but then add their name in the test cases)